### PR TITLE
Refine DARTS tutorial docs

### PR DESCRIPTION
**Pull request by JiahangXu**
### Description ###
Code review for DARTS tutorial

### CODE REVIEW

File: examples/tutorials/darts.py [hunk 1]
```diff
@@ -54,16 +54,13 @@
...
 #    Otherwise, it's optional.
 #
-# When working with famous datasets like CIFAR-10 or ImageNet,
-# it's tempting to use or finetune from a pretrained model, like ResNet.
-# There's nothing wrong with doing so, and sometimes it might be beneficial.
-# Thanks to the development of NAS, we now have quite a large number of *pre-searched models*,
+# NNI presents many built-in model spaces, along with many *pre-searched models* in :doc:`model space hub </nas/space_hub>`,
 # which are produced by most popular NAS literatures.
-# You can easily load these models, validate their performances, and finetune them if you need.
+# Using or finetuning from a pretrained model sometimes might be beneficial in
```
Comment (ultmaster) [inline examples/tutorials/darts.py:59]: in what

File: examples/tutorials/darts.py [hunk 2]
```diff
@@ -54,16 +54,13 @@
...
-# When working with famous datasets like CIFAR-10 or ImageNet,
-# it's tempting to use or finetune from a pretrained model, like ResNet.
-# There's nothing wrong with doing so, and sometimes it might be beneficial.
-# Thanks to the development of NAS, we now have quite a large number of *pre-searched models*,
+# NNI presents many built-in model spaces, along with many *pre-searched models* in :doc:`model space hub </nas/space_hub>`,
 # which are produced by most popular NAS literatures.
-# You can easily load these models, validate their performances, and finetune them if you need.
+# Using or finetuning from a pretrained model sometimes might be beneficial in 
+# A pre-trained model is a saved network that was previously trained on a large dataset like CIFAR-10 or ImageNet.
+# You can easily load these models as a benchmark, validate their performances, and finetune them if you need.
```
Comment (ultmaster) [inline examples/tutorials/darts.py:61]: benchmark?

starting point?

File: examples/tutorials/darts.py [hunk 3]
```diff
@@ -20,7 +20,7 @@
 
 .. _DARTS: https://arxiv.org/abs/1806.09055
 
-Use a pre-searched model
+Use a pre-searched DARTS model
 ------------------------
```
Comment (ultmaster) [inline examples/tutorials/darts.py:24]: Make the `-` long enough.

File: examples/tutorials/darts.py [hunk 4]
```diff
@@ -92,14 +89,14 @@ def evaluate_model(model, cuda=False):
...
 
 # %%
 #
-# The journey could end here. Or you are interested,
+# The journey of using a pre-searched model could end here. Or you are interested,
 # we can go a step further to search a model within :class:`~nni.retiarii.hub.pytorch.DARTS` space on our own.
 #
-# Use the model space
+# Use the DARTS model space
 # -------------------
```
Comment (ultmaster) [inline examples/tutorials/darts.py:100]: Same here

File: examples/tutorials/darts.py [hunk 5]
```diff
@@ -237,11 +238,13 @@ def evaluate_model(model, cuda=False):
 # one-shot strategy combines search with model training into a single run.
 # Compared to multi-trial strategies, one-shot NAS doesn't need to iteratively spawn new trials (i.e., models),
 # and thus saves the excessive cost of model training.
-# It's worth mentioning that one-shot NAS also suffers from multiple drawbacks despite its computational efficiency.
-# We recommend
-# `Weight-Sharing Neural Architecture Search: A Battle to Shrink the Optimization Gap <https://arxiv.org/abs/2008.01475>`__
-# and
-# `How Does Supernet Help in Neural Architecture Search? <https://arxiv.org/abs/2010.08219>`__ for interested readers.
+#
+# .. note::
```
Comment (ultmaster) [inline examples/tutorials/darts.py:242]: I believe a blank line after `.. note::` is required to make it work. But I'm not sure.

File: examples/tutorials/darts.py (hunk 1 above)
Comment (JiahangXu) [reply examples/tutorials/darts.py:59]: my fault. I will delete this sentence

File: examples/tutorials/darts.py (hunk 5 above)
Comment (JiahangXu) [reply examples/tutorials/darts.py:242]: I have tested it and found my way also works. But I will fix this to be consistent with your style.

**Review by ultmaster — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]
looks great!

**Pull request merged**